pwndbg/commands/context.py

```diff
@@ -41,7 +41,6 @@
     "backtrace",
     [
         ColorParamSpec("prefix", "none", "color for prefix of current backtrace label"),
-        ColorParamSpec("address", "none", "color for backtrace (address)"),
         ColorParamSpec("symbol", "none", "color for backtrace (symbol)"),
```

Comment:
i wonder if we should use CODE coloration for symbols too? as is the case in the register output for example:

![image](https://github.com/pwndbg/pwndbg/assets/93072266/feda9768-0033-43f6-bdf2-f72d267a8d56)

Reply:
If we display symbols, probably yes.